extensions/amp-story/1.0/amp-story.js

```diff
@@ -2940,7 +2940,7 @@ export class AmpStory extends AMP.BaseElement {
    */
   maybeLoadStoryDevTools_() {
     if (
-      !getMode().development ||
+      !isModeDevelopment(this.win) ||
```

Comment:
Are we sure we want to reuse all of this code? `#development=actions|amp|amp4ads|amp4email` will also work, and this will be true when tests are running.

Reply:
@choumx From my understanding we want to have a helper that parses the url and returns whether the development mode is active, so we can reuse it easily. What's your suggestion?